src/components/SignInForm.jsx

```diff
@@ -52,9 +52,13 @@ const SignIn = (props, routeProps) => {
 			NameLast
 		} = results;
 
-		props.setFieldValue('NameFirst', NameFirst);
-		props.setFieldValue('NameLast', NameLast);
-		props.setFieldValue('ContactID', contactID);
+		const fields = {
```

Comment:
this can be simplified.

```js
const fields = {
  NameFirst,
  NameLast,
  ContactID: contactID
};
```

ultimately if possible it would be good if `contactID` matched as well, but not blocking, as that could end up being a big rename.

Reply:
basically in es6, if the fields match, than the attribute name will automatically be added to the object

Reply:
That I did not know. I thought I had to let it know the field name. Thats handy. This has been fixed.